overlord/snapstate/refreshhints_test.go

```diff
@@ -323,6 +323,60 @@ func (s *refreshHintsTestSuite) TestRefreshHintsStoresRefreshCandidates(c *C) {
 	c.Check(snapst, DeepEquals, &snapst2)
 }
 
+func (s *refreshHintsTestSuite) TestPruneRefreshCandidates(c *C) {
+	st := s.state
+	st.Lock()
+	defer st.Unlock()
+
+	c.Assert(snapstate.PruneRefreshCandidates(st, "some-snap"), IsNil)
```

Comment:
It's not totally clear to me why you are calling this also at the beginning of the test. Maybe a comment in the code would help :-)

Reply:
Added a comment.